Assets/PlayroomKit/modules/CallbackManager.cs

```diff
@@ -27,14 +27,25 @@ public static string RegisterCallback(Delegate callback, string key = null)
             return key;
         }
 
-
-        public static void InvokeCallback(string key, string arg1 = null)
+        public static void InvokeCallback(string key, string arg1 = null, string arg2 = null)
         {
             if (callbacks.TryGetValue(key, out Delegate callback))
             {
-                if (callback is Action action) action?.Invoke();
-                else if (callback is Action<string> stringAction) stringAction?.Invoke(arg1);
-                else Debug.LogError($"Callback with key {key} is of unsupported type!");
+                switch (callback)
+                {
+                    case Action action:
+                        action?.Invoke();
+                        break;
+                    case Action<string> stringAction:
+                        stringAction?.Invoke(arg1);
+                        break;
+                    case Action<string, string> doubleStringAction:
```

Comment:
Will we have to add another switch statement for every type of argument?

Reply:
Yes, <string, string> is the max we are using in the SDK so far.

Reply:
Does this impact user-provided callbacks? Can the user add any callback which they want?

Reply:
Doesn't impact afaik, data is in string for example RPC sends : data and senderId

Reply:
Can't the data be in number or object format? What if the user sends that? Can we serialize/deserialize it?